mycore-webtools/src/main/vue/processing/src/components/ProcessableComponent.vue

```diff
@@ -30,22 +30,25 @@ function getProgress(model: Processable) {
   return `${model.progress || 0}%`;
 }
 
-function toggleModal() {
-  isModalVisible.value = !isModalVisible.value;
+const processableModal = useTemplateRef("processableModal");
+
+function showModal() {
+  if (processableModal.value) {
+    processableModal.value.show();
+  }
 }
-const isModalVisible = ref(false);
 
 </script>
 
 <template>
   <!-- row -->
-  <div class="table-content">
-    <div class="col col-name">
-      <a href="#" @click.prevent="toggleModal">(P)</a> {{ model.name }}
-    </div>
-    <div class="col col-user">{{ model.user }}</div>
-    <div class="col col-create">{{ Util.formatDate(model.createTime) }}</div>
-    <div class="col col-progress">
+  <tr >
+    <td class="col col-name">
```

Comment:
`td class="col"` looks strange to me
Do not mess with the Bootstrap layout system ;-)